src/webgpu/shader/execution/expression/binary/u32_arithmetic.spec.ts

```diff
@@ -4,10 +4,16 @@ Execution Tests for the u32 arithmetic binary expression operations
 
 import { makeTestGroup } from '../../../../../common/framework/test_group.js';
 import { GPUTest } from '../../../../gpu_test.js';
-import { TypeU32 } from '../../../../util/conversion.js';
-import { fullU32Range } from '../../../../util/math.js';
+import { TypeU32, TypeVec } from '../../../../util/conversion.js';
+import { fullU32Range, sparseF32Range, vectorU32Range } from '../../../../util/math.js';
```

Comment:
I think you want sparseU32Range

Reply:
The callsites below will need to updated also.

Reply:
lol, went through all that trouble to add sparseU32Range, then didn't even use it /facepalm